app/main.py

```diff
@@ -1,6 +1,39 @@
+import os
+from datetime import datetime
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.environ.get("API_KEY")
+API_URL = f"https://api.weatherapi.com/v1/current.json?key={API_KEY}&q="
```

Comment:
If you are using fstrings then this `API_URL` is not a constant any more and will change depending on API_KEY